scripts/pylint_extensions.py

```diff
@@ -757,6 +757,33 @@ def process_module(self, node):
                         'backslash-continuation', line=line_num + 1)
 
 
+class FunctionArgsOrderChecker(checkers.BaseChecker):
+    """Checker for order of arguments. It checks
+       that class arguments comes first.
```

Comment:
De-indent by 4. See other docstrings for example.

Reply:
Sorry for that !